charts/rancher-istio/rancher-istio/100.0.1+up1.10.4/app-readme.md

```diff
@@ -0,0 +1,35 @@
+# Rancher Istio
+
+Our [Istio](https://istio.io/) installer wraps the istioctl binary commands in a handy helm chart, including an overlay file option to allow complex customization. It also includes:
+* **[Kiali](https://kiali.io/)**: Used for graphing traffic flow throughout the mesh
+* **[Jaeger](https://www.jaegertracing.io/)**: A quick start, all-in-one installation used for tracing distributed systemm. This is not production qualified, please refer to jaeger documentation to determine which installation you may need instead.
```

Comment:
```suggestion
* **[Jaeger](https://www.jaegertracing.io/)**: A quick start, all-in-one installation used for tracing distributed system. This is not production qualified, please refer to jaeger documentation to determine which installation you may need instead.
```

Reply:
Bah, I commented on the charts on some of these instead of the package.

Reply:
fixed spelling in latest commits.